addons/mysql/values.yaml

```diff
@@ -3,13 +3,14 @@
 # Declare variables to be passed into your templates.
 
 image:
-  registry: 
-  repository: mysql
+  registry:  registry.cn-hangzhou.aliyuncs.com/apecloud
```

Comment:
Following setting must be consistent for apecloud hosted images:

registry     : docker.io  or  registry.cn-hangzhou.aliyuncs.com
repository: apecloud/mysql-jq